.jenkins/Jenkinsfile

```diff
@@ -90,92 +90,66 @@ def checkDevFlows() {
-        "ACC1804 clang-7 Debug" :               { ACCTest('ACC-1804', 'clang-7', 'SGX1FLC', 'Debug') },
-        "ACC1804 clang-7 RelWithDebInfo" :      { ACCTest('ACC-1804', 'clang-7', 'SGX1FLC', 'RelWithDebinfo') },
-        "Win2016 Debug Cross-platform" :        { win2016DebugCrossPlatform() },
-        "Win2016 Release Cross-platform" :      { win2016ReleaseCrossPlatform() },
-        "Win2016 Debug" :                       { win2016Debug() },
-        "Win2016 Release" :                     { win2016Release() },
-        "ACC1604 Container RelWithDebInfo" :    { ACCContainerTest('ACC-1604') }
+
+parallel "Check Developer Experience" :                { checkDevFlows() },
+         "Sim clang-7 SGX1 Debug" :                    { simulationTest('clang-7', 'SGX1', 'Debug')},
+         "Sim clang-7 SGX1 Release" :                  { simulationTest('clang-7', 'SGX1', 'Release')},
+         "Sim clang-7 SGX1 RelWithDebInfo" :           { simulationTest('clang-7', 'SGX1', 'RelWithDebInfo')},
+         "Sim clang-7 SGX1-FLC Debug" :                { simulationTest('clang-7', 'SGX1FLC', 'Debug')},
+         "Sim clang-7 SGX1-FLC Release" :              { simulationTest('clang-7', 'SGX1FLC', 'Release')},
+         "Sim clang-7 SGX1-FLC RelWithDebInfo" :       { simulationTest('clang-7', 'SGX1FLC', 'RelWithDebInfo')},
+         "ACC1604 clang-7 Debug" :                     { ACCTest('ACC-1604', 'clang-7', 'SGX1FLC', 'Debug') },
+         "ACC1604 clang-7 Release" :                   { ACCTest('ACC-1604', 'clang-7', 'SGX1FLC','Release') },
+         "ACC1604 clang-7 RelWithDebInfo" :            { ACCTest('ACC-1604', 'clang-7', 'SGX1FLC', 'RelWithDebinfo') },
+         "ACC1604 gcc Debug" :                         { ACCTest('ACC-1604', 'gcc', 'SGX1FLC', 'Debug') },
+         "ACC1604 gcc Release" :                       { ACCTest('ACC-1604', 'gcc', 'SGX1FLC', 'Release') },
+         "ACC1604 gcc RelWithDebInfo" :                { ACCTest('ACC-1604', 'gcc', 'SGX1FLC', 'RelWithDebInfo') },
+         "ACC1804 clang-7 Debug" :                     { ACCTest('ACC-1804', 'clang-7', 'SGX1FLC', 'Debug') },
+         "ACC1804 clang-7 RelWithDebInfo" :            { ACCTest('ACC-1804', 'clang-7', 'SGX1FLC', 'RelWithDebinfo') },
+         "Win2016 Debug Cross-platform" :              { win2016CrossPlatform("Debug") },
+         "Win2016 Release Cross-platform" :            { win2016CrossPlatform("Release") },
+         "Win2016 Simulation Debug Cross-platform" :   { win2016CrossPlatform("Debug", 1) },
+         "Win2016 SImulation Release Cross-platform" : { win2016CrossPlatform("Release", 1) },
+         "Win2016 Debug" :                             { win2016("Debug") },
+         "Win2016 Release" :                           { win2016("Release") },
+         "Win2016 Simulation Debug" :                  { win2016("Debug", 1) },
+         "Win2016 Simulation Release" :                { win2016("Release", 1) },
```

Comment:
This is way too much overhead of having 4 more full Windows stages for simulation. Simulation can easily be added to two of the Win2016 runs - the ctest command will need to be run with the simulation flag without having to rebuild everything.

Reply:
We don't need 4 ways of testing simulation on Windows.

Reply:
We don't need 4 ways of testing simulation on Windows - 2 should do.